batchkit_examples/speech_sdk/endpoint_status.py

```diff
@@ -41,6 +41,8 @@ def check_endpoint(self, host: str, port: int, is_secure: bool, is_cloud_service
                     result = json.loads(result.text)
                     if 'valid' in result['apiStatus'].lower() and 'valid' in result['apiStatusMessage'].lower():
                         return True
```

Comment:
All this nonsense should be done by http status code 200 but I recall there being some reason I didn't trust the implementer of the /status